tests/phpunit/tests/block-bindings/register.php

```diff
@@ -11,10 +11,24 @@
  */
 class Tests_Block_Bindings_Register extends WP_UnitTestCase {
 
-	const TEST_SOURCE_NAME       = 'test/source';
-	const TEST_SOURCE_PROPERTIES = array(
-		'label' => 'Test source',
-	);
+	public static $test_source_name       = 'test/source';
+	public static $test_source_properties = array();
+
+	/**
+	 * Set up before each test.
+	 *
+	 * @since 6.5.0
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		self::$test_source_properties = array(
+			'label'              => 'Test source',
+			'get_value_callback' => function () {
```

Comment:
Nit: It would be possible to inject only `get_value_callback` here, but we can change that with the refactoring. Overall, that makes perfect sense to use a static variable 👍🏻